docs/user-guide/system-admin/configuration/installation.md

```diff
@@ -120,8 +114,8 @@ Any other value will be treated as the name of a virus scanner command: the comm
 
 Go to the `<JBOSS>/bin` directory and run
 
-* `standalone.sh` or `start-zanata.sh` for Linux, Mac
-* `standalone.bat` or `start-zanata.bat` for Windows
+* `standalone.sh -c standalone-zanata.xml` or `start-zanata.sh` for Linux, Mac
+* `standalone.bat -c standalone-zanata.xml` or `start-zanata.bat` for Windows
```

Comment:
I think we discussed this before and decided to go with a custom one instead of rewriting the existing one.
I kind of like the fact that now expanding the zip file does not override any files, it just adds new ones. We can change it to rewrite the existing one, or saving a backup of the original one.